pipeline/trusted-resources/pkg/trustedtask/trustedtaskrun_test.go

```diff
@@ -430,18 +448,22 @@ func getSignerFromFile(t *testing.T, ctx context.Context, k8sclient kubernetes.I
 	if err != nil {
 		t.Fatal(err)
 	}
-	cfg := &corev1.ConfigMap{
+
+	cm := &corev1.ConfigMap{
 		ObjectMeta: metav1.ObjectMeta{
 			Namespace: nameSpace,
 			Name:      signingConfigMap,
 		},
-		Data: map[string]string{"signing-secret-path": filepath.Join(tmpDir, "cosign.pub")},
+		Data: map[string]string{signingSecretPath: filepath.Join(tmpDir, "cosign.pub")},
 	}
-	if _, err := k8sclient.CoreV1().ConfigMaps(system.Namespace()).Create(ctx, cfg, metav1.CreateOptions{}); err != nil && !errors.IsAlreadyExists(err) {
-		t.Fatal(err)
+
+	if _, err := k8sclient.CoreV1().ConfigMaps(nameSpace).Update(ctx, cm, metav1.UpdateOptions{}); err != nil {
+		t.Errorf("error updating configmap: %v", err)
 	}
+	// It should be updated by then...
+	time.Sleep(100 * time.Millisecond)
```

Comment:
While this should usually be true, this has the potential to fail / not happen in time. If you use the other ToContext method to set the config directly in the context, it should give you what you want without needing to sleep.